Index: dima/src/main/java/Test.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- dima/src/main/java/Test.java	(revision 4ea4d8183f67b032f988bc6075745ec3be2034a2)
+++ dima/src/main/java/Test.java	(revision )
@@ -1,2 +1,7 @@
 public class Test {
+    public static void main(String[] args) {
+        for (; ; ) {
+            System.out.println("HELLO MR HELLOWORLD");
+        }
+    }
 }
